airflow/models/backfill.py

```diff
@@ -0,0 +1,69 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from sqlalchemy import Boolean, Column, Integer, String, UniqueConstraint
+from sqlalchemy_jsonfield import JSONField
+
+from airflow.models.base import COLLATION_ARGS, ID_LEN, Base
+from airflow.settings import json
+from airflow.utils import timezone
+from airflow.utils.sqlalchemy import UtcDateTime
+
+
+class Backfill(Base):
+    """Model representing a backfill job."""
+
+    __tablename__ = "backfill"
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    dag_id = Column(String(ID_LEN, **COLLATION_ARGS))
```

Comment:
```suggestion
    dag_id = Column(String(ID_LEN, **COLLATION_ARGS))
```

Shouldn't we use [StringID](https://github.com/apache/airflow/blob/b599d81121f3b9103658fba9c33d6f3bebe33cf6/airflow/models/base.py#L84) here instead?

Reply:
sure, done